docs/community/faq/faq-users.md

```diff
@@ -2,69 +2,85 @@
 
-2. How do I install AutoEmulate?
+2. How do I install `AutoEmulate`?
    <!-- Step-by-step instructions on installing the package, including any dependencies that might be required. -->
+   - See the [installation guide](../../getting-started/installation.md) for detailed instructions.
 
-3. What are the prerequisites for using AutoEmulate?
+3. What are the prerequisites for using `AutoEmulate`?
    <!-- Information on the knowledge or data required to effectively use AutoEmulate, such as familiarity with Python, machine learning concepts, or specific data formats. -->
+   - `AutoEmulate` is designed to be easy to use. The user has to first generat a dataset of simulation inputs `X` and outputs `y`, and optimally have a basic understanding of Python and machine learning concepts.
 
 ## Usage Questions
 
-1. How do I start using AutoEmulate with my data?
+1. How do I start using `AutoEmulate` with my simulation?
    <!-- A simple example to get a new user started, possibly pointing to more detailed tutorials or documentation. -->
+   - See the [getting started guide](../../getting-started/quickstart.ipynb) or a more [in-depth tutorial](../../tutorials/01_start.ipynb).
 
-2. What kind of data can I analyze with AutoEmulate?
+2. What kind of data does `AutoEmulate` need to build an emulator?
    <!-- Clarification on the types of datasets suitable for analysis, including data formats and recommended data sizes. -->
 
-3. How do I interpret the results from AutoEmulate?
+   - `AutoEmulate` takes simulation inputs `X` and simulation outputs `y` to build an emulator.`X` is an ndarray of shape `(n_samples, n_parameters)` and `y` is an ndarray of shape `(n_samples, n_outputs)`. Each sample here is a simulation run, so each row of `X` corresponds to a set of input parameters and each row of `y` corresponds to the corresponding simulation output. Currently, all inputs and outputs should be numeric, and we don't support missing data.
+
+   - All models work with multi-output data. We have optimised `AutoEmulate` to work with smaller datasets (in the order of hundreds to thousands of samples). Training emulators with large datasets (hundreds of thousands of samples) may currently require a long time and is not recommended.
+
+3. How do I interpret the results from `AutoEmulate`?
    <!-- Guidance on understanding the output of the software, including any metrics or visualizations it produces. -->
+   - See the [tutorial](../../tutorials/01_start.ipynb) for an example of how to interpret the results from `AutoEmulate`. Briefly, `X` and `y` are first split into training and test sets. Cross-validation and/or hyperparameter optimisation are performed on the training data. After comparing the results from different emulators, the user can evaluate the chosen emulator on the test set with `autoemulate.evaluate_model()`, and plot test set predictions with `autoemulate.plot_model()`. 
```

Comment:
We could link `autoemulate.evaluate_model()` and `autoemulate.plot_model()` here to the API docs maybe?

Reply:
yes, doing that now. Is there a way to directly link to a class method (instead of just the class?)